src/Sentry/PlatformAbstractions/RegistryKeyExtensions.cs

```diff
@@ -0,0 +1,15 @@
+#if NETFX
+using Microsoft.Win32;
+
+namespace Sentry.PlatformAbstractions
+{
+    internal static class RegistryKeyExtensions
+    {
+        public static string GetString(this RegistryKey key, string value)
+            => key.GetValue(value) as string;
```

Comment:
The return type should be nullable or the cast should be replaced with coercion.

Reply:
It compiles only for netfx so no warnings

Reply:
Adding though